app/views/spree/shared/_search.html.erb

```diff
@@ -12,4 +12,11 @@
     <%= search_field_tag :keywords, params[:keywords], placeholder: Spree.t(:search), class: "form-control" %>
   </div>
   <%= submit_tag Spree.t(:search), name: nil, class: "btn btn-success" %>
-<% end %>
+<% end %> -->
+<% @taxons = @taxon && @taxon.parent ? @taxon.parent.children : Spree::Taxon.roots %>
+  <%= form_tag spree.products_path, method: :get do %>
+    <div class="header-search tablet-hide">
+      <%= search_field_tag :keywords, params[:keywords], placeholder: Spree.t(:search), class: "form-control" %>
```

Comment:
Add this text: 'Search...' against this translation key : Spree.t(:search). Please discuss with Umer